bot/exts/moderation/modpings.py

```diff
@@ -85,7 +86,14 @@ async def reschedule_modpings_schedule(self) -> None:
             start_timestamp, work_time = schedule.split("|")
             start = datetime.datetime.fromtimestamp(float(start_timestamp))
 
-            mod = await self.bot.fetch_user(mod_id)
+            guild = self.bot.get_guild(Guild.id)
+            mod = await get_or_fetch_member(guild, mod_id)
+            if not mod:
+                log.info(
```

Comment:
This seems to fit on one-line (I make it out to be 118 chars). If it does, then should be one-line and not wrapped.

Also, I think this should be upgraded to a `log.warning` since it means we likely need to `int e` remove them from the modpings list.